expression/expr_to_pb.go

```diff
@@ -73,7 +74,7 @@ func (pc pbConverter) exprToPB(expr Expression) *tipb.Expr {
 	return nil
 }
 
-func (pc pbConverter) datumToPBExpr(d types.Datum) *tipb.Expr {
+func (pc pbConverter) datumToPBExpr(d types.Datum, ft *types.FieldType) *tipb.Expr {
```

Comment:
I search in this PR for the usage and found that this function only handle Constant. So we could rename it and reduce the two parameter to one *Constant.